pkg/kubelet/kubelet.go

```diff
@@ -884,6 +884,9 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		volumepathhandler.NewBlockVolumePathHandler())
 
 	klet.backOff = flowcontrol.NewBackOff(containerBackOffPeriod, MaxContainerBackOff)
+	klet.backOff.HasExpiredFunc = func(eventTime time.Time, lastUpdate time.Time, maxDuration time.Duration) bool {
+		return eventTime.Sub(lastUpdate) > 600*time.Second
```

Comment:
should the constant be declared next to other constants?

Reply:
It was purposefully changed to inline (though to be fair, not from being a constant but from being a named function like [this](https://github.com/kubernetes/kubernetes/pull/128356/commits/fd46ad0e0e3ca22e9ac5fb2a1c58a7b37a48c51e#diff-67dd9e8f3ee257072765326cb4f242852554a2c0753563fa51e292c0a63a7b94R233) due to [this comment](https://github.com/kubernetes/kubernetes/pull/128356/commits/fd46ad0e0e3ca22e9ac5fb2a1c58a7b37a48c51e#r1819903188)), I don't know if this is a style thing or a Go idiomatic thing or a k8s idiomatic thing so I'm down for whatever.

Reply:
My issue is that some constants are declared in one place, other constant is here. Next thing you will need is to expose it for e2e tests. So you may just move it there already